.travis.yml

```diff
@@ -27,18 +19,46 @@ branches:
 matrix:
     fast_finish: true
     include:
+        - php: 7.1
+          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" DEPENDENCIES="doctrine/instanciator:^1.0.5"
```

Comment:
It's also `instantiator`. Apart from that, any reason why you're locking the version of it? If anything breaks with `--prefer-lowest` because an older version is installed, it needs to be fixed in that package and should probably be added here as a comment so the workaround can be removed.

Reply:
this is adding the dependency for this one case, so its changing composer.json and thus needs a version constraint.

Reply:
I can see that, the question is: why do you need it as an explicit dependency?

Reply:
From Slack: 

> @fbourigault [11:18 AM] 
$ composer why doctrine/instantiator
phpspec/phpspec   2.4.0  requires  doctrine/instantiator (^1.0.1)
phpspec/prophecy  1.4.0  requires  doctrine/instantiator (^1.0.2) (edited)
But it’s for unmaintained versions of those packagesages

Reply:
I see. Thank you for explaining 👍